res/openvpn-gui-res-it.rc

```diff
@@ -327,165 +342,164 @@ Quest'azione può richiedere privilegi amministrativi."
-    IDS_ERR_CREATE_EDIT_LOGWINDOW "Creazione RichEdit LogWindow Fallita!!"
-    IDS_ERR_SET_SIZE "Set Size fallita!"
-    IDS_ERR_AUTOSTART_CONF "Non riesco a trovare una configurazione per partire in automatico: %ls"
-    IDS_ERR_CREATE_PIPE_IN_READ "CreatePipe on hInputRead falito."
-    IDS_NFO_STATE_CONNECTING "Stato corrente: Connessione in corso"
+    IDS_NFO_STATE_SUSPENDED "Stato attuale: sospeso"
+    IDS_ERR_READ_STDOUT_PIPE "Errore in lettura dalla pipe stdout OpenVPN."
+    IDS_ERR_CREATE_EDIT_LOGWINDOW "Creazione RichEdit LogWindow fallita!!"
+    IDS_ERR_SET_SIZE "Impostazione dimensione fallita!"
+    IDS_ERR_AUTOSTART_CONF "Impossibile trovare una configurazione di avvio automatico: %ls"
+    IDS_ERR_CREATE_PIPE_IN_READ "CreatePipe su hInputRead fallito."
+    IDS_NFO_STATE_CONNECTING "Stato attuale: connessione in corso"
     IDS_NFO_CONNECTION_XXX "Connessione OpenVPN (%ls)"
-    IDS_NFO_STATE_CONN_SCRIPT "Stato corrente: Avviato lo script di connessione"
+    IDS_NFO_STATE_CONN_SCRIPT "Stato attuale: avviato script connessione"
     IDS_NFO_STATE_DISCONN_SCRIPT "Stato corrente: Avviato lo script di disconnessione"
     IDS_ERR_RUN_CONN_SCRIPT "Errore durante il funzionamento dello script di connessione: %ls"
-    IDS_ERR_GET_EXIT_CODE "Errore nel ExitCode dello script di connessione (%ls)"
-    IDS_ERR_CONN_SCRIPT_FAILED "Fallito lo script di connessione. (exitcode=%ld)"
-    IDS_ERR_RUN_CONN_SCRIPT_TIMEOUT "Fallito lo script di connessione. TimeOut dopo %d sec."
-    IDS_ERR_CONFIG_EXIST "Esiste già un file di configurazione chiamato '%ls'. Non puoi \
-avere file di configurazioni con lo stesso nome, \
+    IDS_ERR_GET_EXIT_CODE "Errore nell'ExitCode dello script di connessione (%ls)"
+    IDS_ERR_CONN_SCRIPT_FAILED "Esecuzione fallita script di connessione (exitcode=%ld)."
+    IDS_ERR_RUN_CONN_SCRIPT_TIMEOUT "Esecuzione fallita script di connessione. TimeOut dopo %d sec."
+    IDS_ERR_CONFIG_EXIST "Esiste già un file di configurazione '%ls'.\nNon puoi \
+avere più file di configurazioni con lo stesso nome, \
 a meno che non risiedano in cartelle differenti."
-    IDS_NFO_CONN_TIMEOUT "Connessione all'interfacia di gestione fallita.\n\
-Consulta il file di log (%ls) per maggiori dettagli."
+    IDS_NFO_CONN_TIMEOUT "Connessione all'interfaccia di gestione fallita.\n\
+Per maggiori dettagli consulta il file registro (%ls) ."
     /* main - Resources */
     IDS_ERR_OPEN_DEBUG_FILE "Errore nell'apertura del file di debug (%ls) per l'output."
     IDS_ERR_CREATE_PATH "Errore nella creazione del percorso %ls:\n%ls"
-    IDS_ERR_LOAD_RICHED20 "Non riesco a caricare RICHED20.DLL."
-    IDS_ERR_SHELL_DLL_VERSION "La versione del tuo shell32.dll è troppo vecchia (0x%lx). Hai bisogno almeno della versione 5.0."
+    IDS_ERR_LOAD_RICHED20 "Impossibile caricare RICHED20.DLL."
+    IDS_ERR_SHELL_DLL_VERSION "La versione di shell32.dll è troppo vecchia (0x%lx).\nÈ necessaria almeno una versione 5.0."
```

Comment:
As per the resolution in #486, you agreed to change una -> la, but not done.

Reply:
"È necessaria almeno una versione 5.0." in italian is better then ""È necessaria almeno la versione 5.0."
Also I'm not agree I will change it.

Reply:
IMHO using "una" is wrong.

Using "una" makes it sounds as if the user has to pick a version among 5.0.x.
While we are "at least 5.0" so "la" is really indicating that we want that specific version (at least).